test/helpers/test-case.js

```diff
@@ -0,0 +1,27 @@
+const header = 'const test = require(\'ava\');';
+
+function testCaseBuilder(ruleId, options = []) {
+	return function testCase(content, errorMessage = [], withoutHeader = false) {
```

Comment:
`withoutHeader = false` is not very readable at the callsite.

https://ariya.io/2011/08/hall-of-api-shame-boolean-trap

Reply:
Are you talking about the unnamed boolean on call-site, the double negative induce by `withoutHeader = false` or both ?
I can make the use of an object literal to avoid the unnamed option. I personally prefer optional boolean value to be `false`, but I'm ok with the opposite.

Reply:
Both